bigtest/tests/detail-test.js

```diff
@@ -1,17 +1,109 @@
 import { expect } from 'chai';
-import { beforeEach, it } from '@bigtest/mocha';
+import { beforeEach, describe, it } from '@bigtest/mocha';
 import { describeApp } from '../helpers/setup-app';
 
 import DetailPage from '../interactors/detail-page';
+import IndexPage from '../interactors/index-page';
 
 describeApp('Detail Route', () => {
+  let request;
   beforeEach(function() {
-    let request = this.server.create('request');
-    this.visit(`/requests/${request.id}`);
+    request = this.server.create('request', {
+      owner: 'David',
+      status: 'Pending',
+      startDate: '01-01-2019',
+      endDate: '02-01-2019'
+    });
+    return this.visit(`/requests/${request.id}`);
   });
 
   it('has a heading', () => {
     expect(DetailPage.hasHeading).to.equal(true);
     expect(DetailPage.headingText).to.contain('Editing Request');
   });
+
+  it('has fields prepopulated with request values', () => {
+    expect(DetailPage.ownerName).to.equal('David');
+    expect(DetailPage.status).to.equal('Pending');
+    expect(DetailPage.startDate).to.equal('01-01-2019');
+    expect(DetailPage.endDate).to.equal('02-01-2019');
+  });
+
+  it('has a readonly owner name field', () => {
+    expect(DetailPage.nameIsReadOnly).to.be.true;
+  });
+
+  describe('editing input fields', () => {
+    describe('status field', () => {
+      beforeEach(() => {
+        return DetailPage.changeStatus('Denied');
+      });
+
+      it('updates the field', () => {
+        expect(DetailPage.status).to.equal('Denied');
+      });
+    });
+    describe('start date field', () => {
```

Comment:
When reading a test case, the `describe` blocks when strung together with the it blocks should read as a sentence.

'editing input fields start date field updates the field` doesn't make sense. It probably works best to have the outer level describe just say "editing"

Reply:
That way it would read: "editing the start date field updates the start date"